packages/postman-bin/postman-bin.pacscript

```diff
@@ -0,0 +1,35 @@
+name="postman-bin"
+version="9.24.2"
+description="Build, test, and document your APIs faster"
+repology=("project: postman" "visiblename: postman-bin")
+url="https://kaifa.ch/pacstall/postman-${version}-linux-x64.tar.gz"
+hash="cfb85d7ae366ee1487ce1b2dab1c19ffa25c214af5c083a09447ed488493b115"
+maintainer="Marie Piontek <marie@kaifa.ch>"
+gives="postman"
+breaks="postman postman-deb postman-app"
+
+depends="libxss1 libnss3 libgtk-3-0"
+
+prepare() {
+    true
+}
+
+build() {
+    true
+}
+
+install() {
+    # Get desktop file and install it
+    wget -q "https://aur.archlinux.org/cgit/aur.git/plain/postman.desktop?h=postman-bin" -O postman.desktop
+    sudo install -Dm644 postman.desktop -t "${STOWDIR}/${name}/usr/share/applications"
+
+    # Install the rest of the app
+    sudo install -d "${STOWDIR}/${name}/opt/postman/" "${STOWDIR}/${name}/usr/bin"
+    sudo install -d "${STOWDIR}/${name}/usr/share/icons/hicolor/128x128/apps"
+
+    sudo cp -r ${SRCDIR}/Postman/app/* ${STOWDIR}/${name}/opt/postman/
```

Comment:
Quote variables

Reply:
I wanted to name the commit where i quote them: "no-fix: haha pacscript go broke" but decided for . in the end cause i knew it would break if i quote them as this also happened with tidal-hifi back then.

Reply:
as you can't quote something with * in it or else it expects * to be an folder or file